### DISCUSSION TITLE
Homematic devices show no response in HomeKit

### DISCUSSION THREAD

**Discussion by gethash** [SukramJ/aiohomematic#777] [General]
**custom_component version (if applicable):**
1.25.1

**known last working custom_component version (if applicable):**
perhaps 1.24.4 but i am not sure

**Home Assistant:**
- Version: 2022.x.x
- Installation method (HAOS, supervised, core, venv, Docker): 

The problem ist not directly related to hahomematic, but i thought I give it a short and perhaps other people ran into that problem too. I just have 3 home-magic devices. A garage control (hörmann) and a switch. All these components just work fine in home assistant but show no response in HomeKit (Apple home). All other devices (shelly, knx and so on) work just finde in HomeKit, also i cannot find any error in the debug logs and updated everything to the latest version. Thanks in advance for any ideas...

**Comment by SukramJ** [comment]
Moved this to discussions because the issue tracker is for issues only.

**Reply by gethash** [reply]
You are right. Sorry and thanks for moving it over.

**Comment by SukramJ** [comment]
Which device (HmIP-XXX) do you use? Which entities do not respond?

**Comment by gethash** [comment]
for IP i am using an HmIP-MOD-HO. Again, it works fine in home assistant itself but the export in HomeKit does not work. Frankly i would not think that this is an homematic problem, but I was wondering since all other components work fine in home assistant and HomeKit.

**Comment by gethash** [comment]
It turns out that there was an error just with this HomeKit component. Recreating the bridge and adding it to HomeKit again solved the problem.
